CHANGELOG.md

```diff
@@ -22,6 +22,12 @@ is owned by that user.
 
 The [1.5 release](docs/1_5_Upgrade.md) is scheduled for early July. 
 
+## 1.4.6
+
+### Bug Fixes
+
+- Fixed case where networks using onchain permissioning could stall when the bootnodes were not validators. [\#969](https://github.com/hyperledger/besu/pull/969)
```

Comment:
do we need to mention IBFT?

Reply:
I don't think so.  The code isn't specific to IBFT and the known issue didn't reference it either.